service/awsconfig/v10/resource/cloudformation/spec.go

```diff
@@ -19,10 +21,17 @@ type StackState struct {
 	MasterInstanceResourceName string
 	MasterCloudConfigVersion   string
 
+	ShouldScale  bool
+	ShouldUpdate bool
+
+	Status string
+
 	WorkerCount              string
 	WorkerImageID            string
 	WorkerInstanceType       string
 	WorkerCloudConfigVersion string
 
+	UpdateStackInput cloudformation.UpdateStackInput
```

Comment:
We have to extend the stack state with a couple of information. We should dispatch the same structure within a resource implementation. This is still not really straight here but I tried to make it better.